test/powershell/enginecore/Eventing.Tests.ps1

```diff
@@ -0,0 +1,36 @@
+Describe "BooleanParameterDCR Tests" -tags "CI" {
+    BeforeEach {
+        Get-EventSubscriber | Unregister-Event
+    }
+    AfterEach {
+        Get-EventSubscriber | Unregister-Event
+    }
+
+    # can't let this case to work, need help
+    It "Register an event with no action, trigger it and wait for it to be raised." -Pending:$true{
+        Get-EventSubscriber | should BeNullOrEmpty
+        $messageData = new-object psobject
+        $job = Start-Job { Start-Sleep -Seconds 5; 1..5 }

```

Comment:
BTW, I need help that "After new-event EventSIDTest, I cannot get-event EventSIDTest, it shows 'get-event : Event with source identifier 'EventSIDTest' does not exist.'", could you help me on this? I think this will be not effected by the job failed, right?